src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,212 @@
+            >
+              Incorporate Now
+            </v-btn>
+          </div>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { CompanyType } from '@/enums'
+import NameInput from '@/components/new-request/name-input.vue'
+import { Navigate } from '@/plugins'
+import BusinessServices from '@/services/business.services'
+import { BusinessRequest } from '@/interfaces'
+
+@Component({
+  components: {
+    NameInput
+  }
+})
+export default class BulletsColinLink extends Vue {
+  /** The selected business type. */
+  @Prop({ default: '' }) readonly businessType!: string
+
+  /** Show Colin Button. */
+  @Prop({ default: false }) readonly colinButton!: boolean
+
+  /** Show Designation Dropdown. */
+  @Prop({ default: false }) readonly showDesignation!: boolean
+
+  /** Whether user is authenticated. */
+  @Getter getIsAuthenticated!: boolean
+
+  // Local properties
+  selectedCompanyType = ''
+  bulletPoints = [
+    'Your business name will be the Incorporation Number assinged by the Registry.',
+    'You can change your business name at a later date.',
+    'It is not possible to request a specific Incorporation Number.'
+  ]
+  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
+
+  // For template
+  readonly CompanyType = CompanyType
+
+  // Get radio text respective to Enum values of CompanyType.
+  getRadioText (option: string): string {
+    if (option === CompanyType.NAMED_COMPANY) {
+      return `Named Company`
+    } else if (option === CompanyType.NUMBERED_COMPANY) {
+      return 'Numbered Company'
+    }
+    return '[error]'
+  }
+
+  /**
+   * The alternate short names for entity types.
+   * Alternate names are used in Entities UIs.
+   */
+  entityTypeAlternateShort (entityType: string): string {
+    if (entityType === 'BC' || entityType === 'CC') return entityType
+    if (entityType === 'CR') return 'BEN'
+    if (entityType === 'UL') return 'ULC'
+    return ''
+  }
+
+  // Navigate to the Entity Dashboard.
+  goToEntityDashboard (businessId: string): void {
+    if (businessId) {
+      const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
+      Navigate(`${dashboardUrl}${businessId}`)
+    }
+  }
+
+  /**
+   * If user is authenticated, create draft business and redirect to Dashboard.
+   * If user is not authenticated, redirect to login screen then redirect back.
+   */
+  async incorporateNow () {
+    if (this.getIsAuthenticated) {
+      try {
+        // show spinner since this is a network call
+        this.$root.$emit('showSpinner', true)
+        const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
+        const businessId = await this.createBusiness(accountId)
+        this.goToEntityDashboard(businessId)
+      } catch (error) {
+        this.$root.$emit('showSpinner', false)
+        throw error
```

Comment:
You should show the error here. Throwing it, with nobody to catch it, will display something in the console, but the user won't otherwise know that something went wrong.

Reply:
I think this got fixed when you moved `incorporateNow()` to nr-affiliation-mixin.ts.

Reply:
Actually, I'm not sure. Does error handling work now?

Reply:
No Sev, it does not. I still need to fix it. Throwing a new Error is still showing it in the console. I may have to use the Error dialog.